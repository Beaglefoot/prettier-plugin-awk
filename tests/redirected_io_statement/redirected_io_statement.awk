{
    print "bad">"/dev/stderr"
    printf("bad")>>"/dev/stderr"
}
