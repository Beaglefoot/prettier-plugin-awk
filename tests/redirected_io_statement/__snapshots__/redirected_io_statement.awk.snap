{
    print "bad">"/dev/stderr"
    printf("bad")>>"/dev/stderr"
}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
{
    print "bad" > "/dev/stderr"
    printf("bad") >> "/dev/stderr"
}
